adapters/repos/db/shard_test.go

```diff
@@ -850,3 +750,10 @@ func TestShard_resetDimensionsLSM(t *testing.T) {
 	require.Nil(t, idx.drop())
 	require.Nil(t, os.RemoveAll(idx.Config.RootPath))
 }
+
+func getVectorIndexAndQueue(t *testing.T, shard ShardLike, targetVector string) (VectorIndex, *VectorIndexQueue) {
+	idx, vok := shard.GetVectorIndex(targetVector)
```

Comment:
```suggestion
        t.Helper()
	idx, vok := shard.GetVectorIndex(targetVector)
```

suggestion: this function makes assertions that may fail, `t.Helper()` produces better stack traces in the error log

Reply:
I see you are a big fan of these `t.Helper()` :D I don't see much value here as the stack trace is 2-3 levels deep anyway, so it is very easy to understand where is the error coming from, especially with IDE highlight. And these helper calls take screen real estate

Reply:
> I see you are a big fan of these t.Helper() :D

Yeah, I was really thinking if I should add this suggestion here and if it wasn't just my [golden hammer](https://en.wikipedia.org/wiki/Law_of_the_instrument) of sorts :sweat_smile:

My rule of thumb's been to mark a function as `t.Helper` if it receives `*testing.T` _can fail a test_. But I haven't really seen a definitive guideline on that, so it's mostly a personal preference.